plume-models/src/medias.rs

```diff
@@ -202,10 +204,9 @@ impl Media {
             .next()
             .map(ToOwned::to_owned)
             .unwrap_or_else(|| String::from("png"));
-        let path =
-            Path::new("static")
-                .join("media")
-                .join(format!("{}.{}", GUID::rand().to_string(), ext));
+        let path = Path::new(&super::CONFIG.media_directory)
+            .join("media")
```

Comment:
the `media` part is already included in the default value of media_directory (`static/media`), and is, I think, not required for a custom media directory